src/main/java/io/seqera/tower/cli/responses/ComputeEnvs/ComputeEnvExport.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.responses.ComputeEnvs;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import io.seqera.tower.JSON;
+import io.seqera.tower.cli.responses.Response;
+import io.seqera.tower.cli.utils.FilesHelper;
+import io.seqera.tower.model.ComputeConfig;
+
+public class ComputeEnvExport extends Response {
+
+    public final ComputeConfig computeConfig;
+    public final String fileName;
+
+    public ComputeEnvExport(ComputeConfig computeConfig, String fileName) {
+        this.computeConfig = computeConfig;
+        this.fileName = fileName;
+    }
+
+    @Override
+    public String toString() {
+        String configOutput = "";
+
+        try {
+            configOutput = new JSON().getContext(ComputeConfig.class).writerWithDefaultPrettyPrinter().writeValueAsString(computeConfig);
+        } catch (JsonProcessingException e) {
+            e.printStackTrace();
+        }
+
+        if (fileName != null && !fileName.equals("-")) {
+            FilesHelper.saveString(fileName, configOutput);
```

Comment:
The save of the file has to be done at the `ExportCmd` instead of here. The idea is that the responses only do presentation, no business logic. In fact, the `toString` method it is not call when the user wants always `json` output. (when using `tw --json ...`)

Reply:
To me, a file is a way to present data. Under that criteria, should we move the JSON conversion to the command as well?

Reply:
Yes, you're right. The JSON conversion also needs to be moved to the command as well.